gpu_report.rst

```diff
@@ -333,6 +333,10 @@ A possibly faster form of the previous loop is shown below.
 The ``simd`` directs the team to use SIMD-like instructions over the threads.
 This is almost always the default behavior, so it is often omitted.
 
+Note that as of `9th April, 2025` AMD compilers don't understand the `loop`
+directive. So, for portability, it is probably preferable to use the less-concise
+`!$omp teams distribute parallel do simd ...` directive.
```

Comment:
I think for now it's still OK to adopt modern constructs supported by the platform that matters for us (Nvidia), but we should start gathering feedback to send back to AMD.

But if it looks like we won't get much action from them, and the AMD accelerators prove viable, then we probably will need to adopt the heavier form.